.gitignore

```diff
@@ -23,4 +23,4 @@ package-lock.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-/src/config/firebase-config.ts
```

Comment:
Removing firebase config ignore line cos now we'll take advantage of it, as seen in the change below.